examples/models/llama/config/llm_config_utils.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+# Copyright 2025 Arm Limited and/or its affiliates.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+
+from executorch.examples.models.llama.config.llm_config import LlmConfig
+
+
+def convert_args_to_llm_config(args: argparse.Namespace) -> LlmConfig:
```

Comment:
I think this should be a static method of `LlmConfig`. For example `LlmConfig.from_args()`